arangodb-net-standard/CursorApi/CursorResponseExtra.cs

```diff
@@ -1,11 +1,34 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using System.Collections.Generic;
 
 namespace ArangoDBNetStandard.CursorApi
 {
     public class CursorResponseExtra
```

Comment:
I don't see `warnings` in this class. Was it excluded on purpose?

Reply:
No. Can you indicate where you see it documented?